install/operator/pkg/syndesis/configuration/configuration.go

```diff
@@ -472,16 +475,31 @@ func metaVolumeCapacityFromEnv(config map[string]string, syndesis *v1alpha1.Synd
 }
 
 func envUpgradeVolumeCapacity(syndesis *v1alpha1.Syndesis) *SyndesisEnvVarConfig {
-    if capacity := syndesis.Spec.Components.Upgrade.Resources.VolumeCapacity; capacity != "" {
-        return &SyndesisEnvVarConfig{
-            Var:   EnvUpgradeVolumeCapacity,
-            Value: capacity,
-        }
-    }
-    return nil
+	if capacity := syndesis.Spec.Components.Upgrade.Resources.VolumeCapacity; capacity != "" {
+		return &SyndesisEnvVarConfig{
+			Var:   EnvUpgradeVolumeCapacity,
+			Value: capacity,
+		}
+	}
+	return nil
 }
 func upgradeVolumeCapacityFromEnv(config map[string]string, syndesis *v1alpha1.Syndesis) {
-    if v, ok := getString(config, EnvUpgradeVolumeCapacity); ok {
-        syndesis.Spec.Components.Upgrade.Resources.VolumeCapacity = v
-    }
+	if v, ok := getString(config, EnvUpgradeVolumeCapacity); ok {
+		syndesis.Spec.Components.Upgrade.Resources.VolumeCapacity = v
+	}
+}
+
+func envExposeVia3Scale(syndesis *v1alpha1.Syndesis) *SyndesisEnvVarConfig {
+	return &SyndesisEnvVarConfig{
+		Var:   EnvExposeVia3Scale,
+		Value: strconv.FormatBool(syndesis.Spec.Components.Server.ExposeVia3Scale),
+	}
+}
+
+func exposeVia3Scale(config map[string]string, syndesis *v1alpha1.Syndesis) {
+	if v, ok := getString(config, EnvExposeVia3Scale); ok {
+		if b, err := strconv.ParseBool(v); err != nil {
+			syndesis.Spec.Components.Server.ExposeVia3Scale = b
```

Comment:
Maybe we introduce here an intermediate level 'Features' ? (`syndesis.Spec.Components.Server.Features.ExposeVia3Scale`)